### DISCUSSION TITLE
Character grid is always centered inside the window

### DISCUSSION THREAD

**Discussion by apaz-cli** [kovidgoyal/kitty#5636] [Q&A] [answered]
As the window is resized, it seems that kitty works out how many characters it can fit into those dimensions, and then centers the space where it renders those characters inside the window. In other words, half the vertical unused space is at the top, and half is at the bottom. The same is true of left and right, but the effect is less pronounced.

![centered](https://user-images.githubusercontent.com/47389403/199130717-915b46c7-b6b4-468c-a341-0029271dcfb4.png)

Other terminals either only resize to a multiple of the number of pixels required to render a character, or always render starting from the top left. Is it possible to configure this? While this isn't an important feature, a lot of terminal applications look bad with unused space at the top and bottom.

**Comment by kovidgoyal** [comment, marked as answer]
Both of those are configurable

https://sw.kovidgoyal.net/kitty/conf/
